sample-compose/app/src/test/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -106,10 +106,10 @@ class HomeScreenTest : BaseScreenTest() {
 
     private fun initViewModel() {
         viewModel = HomeViewModel(
-            coroutinesRule.testDispatcherProvider,
             mockGetModelsUseCase,
             mockIsFirstTimeLaunchPreferencesUseCase,
-            mockUpdateFirstTimeLaunchPreferencesUseCase
+            mockUpdateFirstTimeLaunchPreferencesUseCase,
```

Comment:
Can we add the test case for the `FirstTimeLaunch`?

Reply:
Updated in https://github.com/nimblehq/android-templates/pull/478/commits/d8c44c06ca24799ca04afcc1b59913dd1c4725d9

Reply:
@AVI5HEK, What about in `HomeScreenTest`?

Reply:
@Wadeewee done. https://github.com/nimblehq/android-templates/pull/478/commits/2781508896349644162e42e186fa703ad23dd090